src/main/proto/th2_grpc_data_provider/data_provider_template.proto

```diff
@@ -264,6 +281,7 @@ message MessageSearchRequest {
 
   repeated MessageID message_id = 8; // List of message IDs to restore the search. If given, it has the highest priority and ignores 'stream' (uses streams from the ids), 'start_timestamp' and 'resume_fromId'. Set by default to null.
 
+  google.protobuf.BoolValue attached_events = 10; // If true, it will additionally load attachedEventsIds. Set by default to false.

```

Comment:
Why don't we want to use `bool` type instead? Do we really need to know if the value was not set?

Reply:
This is done for consistency with the HTTP API, where if the field is not specified then false or true is set by default. Here I also did so that it was possible not to rely on the default values of the protobuf itself, but to expose your own